src/nodedef.cpp

```diff
@@ -1069,6 +1070,36 @@ content_t CNodeDefManager::allocateDummy(const std::string &name)
 }
 
 
+void CNodeDefManager::unallocateName(const std::string &name)
+{
+	// Pre-condition
+	assert(name != "");
+
+	// Erase name from name ID mapping
+	content_t id = CONTENT_IGNORE;
+	if (m_name_id_mapping.getId(name, id)) {
+		m_name_id_mapping.eraseName(name);
+		m_name_id_mapping_with_aliases.erase(name);
+	}
+
+	// Erase node content from all groups it belongs to
+	for (std::map<std::string, GroupItems>::iterator i = m_group_to_items.begin();
+		i != m_group_to_items.end() ; ++i) {
```

Comment:
This needs to be indented an extra level because it's a control structure continuation.